chart/README.md

```diff
@@ -520,12 +521,12 @@ Some example commands are shown below.
 #### External Database, Dynamic EFS, Dynamic ALB, and Secrets
    ```
    helm install mycluster
-   --set iq_server.serviceAccountName="nxlc-default-d95tfx8l-service"
+   --set iq_server.serviceAccountName=nxlc-service-account-name \
```

Comment:
To follow the same pattern we've used earlier and I wouldn't include line endings since it can be different in different consoles
```suggestion
   --set iq_server.serviceAccountName=<service account name, default "default">
```